platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -118,6 +118,10 @@ The `testResourcesDirs` and `testSourcesDirs` properties in `org.gradle.plugins.
 Gradle now emits deprecation warnings when these properties are used.
 They will be removed in Gradle 9.0.
 
+==== `ForkOptions.getJavaHome()` and `ForkOptions.setJavaHome()` are no longer deprecated
```

Comment:
🤔 Maybe add a note that references this one on the original deprecation statement?

That would help user trying to move away from it to realise they don't have to if they upgrade to 8.14.

Reply:
That's not a bad idea.  Done.